src/components/ec-donut/ec-donut.vue

```diff
@@ -101,92 +116,92 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/colors/index';
-@import '../../scss/tools/';
-
+<style>
 .ec-donut {
-  display: flex;
-  flex-wrap: wrap;
+  @apply tw-flex;
+  @apply tw-flex-wrap;
 
   &__container {
-    margin-right: 24px;
+    @apply tw-mr-24;
+
     height: 108px;
 
-    @include media__up-to-sm {
-      width: 100%;
-      margin-right: 0;
-      display: flex;
-      justify-content: center;
+    @media screen and (max-width: theme('screens.sm')) {
```

Comment:
We are not following mobile first. Don't do max-width in `@media`

Make those CSS properties default and write a new CSS what should happen when screen is bigger than `sm`
```
&__container {
  @apply tw-w-full;
  @apply tw-mr-0;
  ...

  @screen sm { /* or md, I don't know */
     @apply tw-block;
     ...
  }
}
```